app/services/email_service.rb

```diff
@@ -1,20 +1,38 @@
-# :nocov:
 class EmailService
   # rubocop:disable Metrics/MethodLength
   def self.send_allocation_email(params)
     offender = OffenderService.get_offender(params[:nomis_offender_id])
     pom = PrisonOffenderManagerService.get_pom(params[:prison], params[:nomis_staff_id])
-    last_allocation = Allocation.where(nomis_offender_id:  params[:nomis_offender_id]).
-        where(active: false).last
+    last_allocation = Allocation.where(
+      nomis_offender_id: params[:nomis_offender_id],
+      active: false
+    ).last
     message = params[:message]
+    url = Rails.configuration.allocation_manager_host + '/poms/my_caseload'
```

Comment:
Is there a *_path() we could use instead of hard-coding?

Reply:
Tried using `my_caseload_path`, but it's not recognised, think we might have to hardcode it

Reply:
Rails.application.routes.url_helpers.my_caseload_oath